Shared/Samples/Add dynamic entity layer/AddDynamicEntityLayerView.SettingsView.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+extension AddDynamicEntityLayerView {
+    struct SettingsView: View {
+        /// The view model for the sample.
+        @EnvironmentObject private var model: Model
+        
+        var body: some View {
+            List {
+                Section("Track display properties") {
+                    Toggle("Track lines", isOn: $model.showsTrackLine)
```

Comment:
If I turn "Previous observations" toggle on, it seems the track lines setting is overridden in the graphics, only shows the history blue dots, not the line. I'm assuming it is expected behavior.

Reply:
I'm not seeing that.  I think if you zoom in on a track you can see the lines.  When zoomed out, the lines are obscured by the dots.